models/User.js

```diff
@@ -0,0 +1,77 @@
+const mongoose = require('mongoose');
+const validator = require('validator');
+const bcrypt = require('bcryptjs');
+
+const UserSchema = new mongoose.Schema({
+	firstName: {
+		type: String,
+		required: [true, 'Please enter your first name'],
+		minlength: 3,
+		maxlength: 30,
+	},
+	lastName: {
+		type: String,
+		required: [true, 'Please enter your last name'],
+		minlength: 3,
+		maxlength: 100,
+	},
+	email: {
+		type: String,
+		unique: true,
+		required: [true, 'Please provide email'],
+	},
+	password: {
+		type: String,
+		unique: true,
+		required: [true, 'Please provide email'],
+		minlength: 8,
```

Comment:
Why would this ever have `minlength`? Pure SHA256 is already 64 chars, let alone salt.

Reply:
Removed :)